src/features/klant/KlantAanmaken.vue

```diff
@@ -109,11 +109,20 @@ const contactmomentStore = useContactmomentStore();
 
 const savingKlant = ref(false);
 
+const formWarning = ref("");
```

Comment:
als je van de formWarning een computed maakt, dan verdwijnt hij direct vanzelf weer zodra je de benodigde gegevens in het formulier invult. bij het submitten hoef je dan alleen te checken of formWarning een waarde heeft
zoiets
```
const formWarning= computed(() =>
 return conditie ? "bla bla" : ""
);
```